src/wlapplication_messages.cc

```diff
@@ -19,177 +19,236 @@
 
 #include "wlapplication_messages.h"
 
+#include <algorithm>
 #include <iostream>
+#include <regex>
 
 #include <boost/format.hpp>
 
 #include "base/i18n.h"
 
 using std::endl;
 
+static size_t kIndent = 23;
+
+#ifndef _WIN32
+static std::string kDefaultHomedir = "~/.widelands";
+#else
+static std::string kDefaultHomedir = "%USERPROFILE%\\.widelands";
+#endif
+
+/// Command line help
+/// Title: unindented text in the line above
+/// Key: the actual parameter
+/// Hint: text after =
+/// Help: Full text help
+/// Verbose: Filter some config options (--help vs. --help-all)
+/// Note that the _() markup is only for extraction. Since no textdomain is available in
+/// static context these should be marked again on usage.
```

Comment:
That's bad, since it attempts to fetch the strings from an arbitrary textdomain and localedir and creates according entries in gettext's string cache. Better wrap either the vector or the localized entries in a function that sets the correct textdomain.